### PR TITLE
Mobile screen width "collapsed" tab styling

### LINKED ISSUE #4245: Inconsistency in tab display on mobile view
While testing out mobile dropdown, it seems like after a certain point, the buttons on /notes and /questions but not /tags stop displaying text, most likely for better ui on mobile view. Also it happens when the navbar takes up extra vertical space.
ezgif com-video-to-gif

Perhaps the bug with tabs is a result of another bug of text reappearing at that aspect ratio?

The best resolution for this bug would probably be:

- [ ] prevent navbar from increasing in height
- [x] allow tabs on /tags to turn to just icons too
- [ ] prevent tabs on all three pages from adding back text when the window continues to shrink

Some files to look into:
/app/views/notes/index.html.erb
/app/views/questions/index.html.erb

### PR DESCRIPTION
**Pull request by JonathanXu1**
Fixes #4245

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

Small icons designed for mobile-friendly view now work properly. Gif demonstrating this on /notes, /tag, and /questions:
ezgif com-video-to-gif 1

### CODE REVIEW

**Comment by SidharthBansal** [comment]
Please provide a better screenshot and then claim the task

**Comment by SidharthBansal** [comment]
https://codein.withgoogle.com/dashboard/tasks/4640364225888256/

**Comment by jywarren** [comment]
This looks good but the icons are touching the text -- are you able to add a space between icon and text so that the design is a little tidier? Thank you!

File: app/views/tag/show/_nav_tabs.html.erb [hunk 1]
```diff
@@ -1,27 +1,48 @@
 <ul class="nav nav-tabs" style="margin-top:10px;">
   <!-- when will it not be show...? -->
   <% if params[:action] == "show" %>
-    <% unless params[:id].match("question:") %><li<% if @node_type == "note" %> class="active"<% end %>><a href="/tag/<%= params[:id] %>"><i class="fa fa-file"></i> <%= raw t('tag.show.research_notes') %></a></li><% end %>
+    <% unless params[:id].match("question:") %><li<% if @node_type == "note" %> class="active"<% end %>><a href="/tag/<%= params[:id] %>"><i class="fa fa-file"></i><span class="hidden-sm hidden-xs"><%= raw t('tag.show.research_notes') %></span></a></li><% end %>
```
[the remark this answers is not in the document]
Comment (JonathanXu1) [reply app/views/tag/show/_nav_tabs.html.erb]: Alright I'll push the changes with the spaces!

**Comment by JonathanXu1** [comment]
@jywarren sorry about the delay, I was busy with schoolwork. I've just pushed changes that applied the spaces between the icons and text.

**Comment by JonathanXu1** [comment]
It's applied to tags, notes, and questions, like so:
image

**Comment by jywarren** [comment]
Awesome. I'm in a train but this looks good to go, ready to merge. Thanks!!!

**Pull request merged**